Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json

```diff
@@ -1188,7 +1049,7 @@
         ]
     },
     "group": "incident",
-    "id": "Phishing Incident v3",
+    "id": "Phishing Incident_copy",
```

Comment:
id should not be changed

Reply:
[SDK bug](https://github.com/demisto/etc/issues/41253), I'll fix manually